src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -180,6 +180,15 @@
   <data name="DoNotDisableSchUseStrongCryptoMessage" xml:space="preserve">
     <value>{0} disables TLS 1.2 and enables SSLv3</value>
   </data>
+  <data name="DoNotInstallRootCert" xml:space="preserve">
+    <value>Do Not Add Certificate To Root Store</value>
+  </data>
+  <data name="DoNotInstallRootCertDescription" xml:space="preserve">
+    <value>By default, the Trusted Root Certification Authorities certificate store is configured with a set of public CAs that has met the requirements of the Microsoft Root Certificate Program. Since all trusted root CAs can issue certificates for any domain, an attacker can pick a weak or coercible CA that you install by yourself to target for an attack – and a single vulnerable, malicious or coercible CA undermines the security of the entire system. To make matters worse, these attacks can go unnoticed quite easily.</value>
+  </data>
+  <data name="DoNotInstallRootCertMessage" xml:space="preserve">
+    <value>It is not recommended to install additional certificate into your operating system's Root certificate store/keychain</value>
```

Comment:
>It is not recommended to install additional certificate into your operating system's Root certificate store/keychain [](start = 11, length = 116)

Adding certificates to the operating system's trusted root certificates increases the risk of incorrectly authenticating an illegitimate certificate.